hax/hax/motr/hax.c

```diff
@@ -119,6 +119,20 @@ static void entrypoint_request_cb(struct m0_halon_interface *hi,
 				  bool first_request)
 {
 	struct hax_entrypoint_request *ep;
+	struct hax_link               *hxl;
+
+	M0_ALLOC_PTR(hxl);
+	if (hxl == NULL) {
+		M0_LOG(M0_ERROR, "Cannot allocate hax_link");
+		return;
+	}
+	hxl->hxl_req_id = *req_id;
+	hxl->hxl_ep_addr[0] = '\0';
+	strncat(hxl->hxl_ep_addr, remote_rpc_endpoint, EP_ADDR_BUF_SIZE - 1);
```

Comment:
[* style *] s/EP_ADDR_BUF_SIZE/ARRAY_SIZE(...)/ to avoid hardcoding.

Reply:
I don't see hardcoding here. If the endpoint address buffer size will ever be change - it will pick up the new value here automatically. Moreover, `EP_ADDR_BUF_SIZE` is shorter than `ARRAY_SIZE(hxl->hxl_ep_addr)` and easier to read.

Reply:
If someone changes `hxl_ep_addr` size, but doesn't change the constant the code will break. My point is that the size of the array is defined in 2 different places (here and in .h) and there is an easy way to avoid this.

Reply:
Sorry, I don't understand this. The `EP_ADDR_BUF_SIZE` is defined at one place only. And it is used to define the size of `hxl_ep_addr`.

Reply:
EP_ADDR_BUF_SIZE is defined in one place, but there are 2 places where the size of the array is used as `EP_ADDR_BUF_SIZE`: here and in `.h`.